acceptance/update_test.go

```diff
@@ -86,7 +90,9 @@ func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig
 	// of the last node receiving the freeze command).
 	// Note that this is the freeze trigger stalling on the Replica, not the
 	// Store-polling mechanism.
-	if reply, err := postFreeze(c, true); !testutils.IsError(err, "timed out waiting for Range|Timeout exceeded while") {
+	acceptErrs := strings.Join([]string{"timed out waiting for Range",
```

Comment:
I see your point, but given that this test fails pretty reliably on master, I don't want to invest another CI period.